placesautocomplete/src/main/res/values/pacv_attrs.xml

```diff
@@ -5,11 +5,13 @@
         <attr name="pacv_googleMapsApiKey" format="string"/>
         <attr name="pacv_adapterClass" format="string"/>
         <attr name="pacv_historyFile" format="string"/>
+        <attr name="pacv_languageCode" format="string"/>
```

Comment:
To remain consistent with the other parameters that are settable by attribute, there should be equivalent java methods on `PlacesAutocompleteTextView` for setting the language value.